parsing/dialects/dialect1/src/test/java/org/apache/calcite/test/Dialect1ParserTest.java

```diff
@@ -2674,4 +2674,22 @@ private void checkExNotNull(SqlParserUtil.StringAndPos sap, Throwable thrown) {
     final String expected = "(?s).*Encountered \"\\( -\" at .*";
     sql(sql).fails(expected);
   }
+
```

Comment:
Just to make sure. Are we adding this feature to all parser instead of only adding it to the Dialect 1? If so, it might be more useful to put the tests in the core parser test instead.

Reply:
This is just for Dialect1 since postExpressionMethods is only filled in the Dialect1 config.fmpp